tests/create_LWE.py

```diff
@@ -0,0 +1,242 @@
+from addict import Dict
+
+from CADETProcess.simulator import Cadet
+
+from CADETProcess.processModel import (
+
+    ComponentSystem,
+    FlowSheet,
+
+    Inlet,
+    Outlet,
+    Cstr,
+    GeneralRateModel,
+    TubularReactor,
+    LumpedRateModelWithoutPores,
+    LumpedRateModelWithPores,
+    MCT,
+
+    Linear,
+    Langmuir,
+    LangmuirLDF,
+    BiLangmuir,
+    BiLangmuirLDF,
+    FreundlichLDF,
+    StericMassAction,
+    AntiLangmuir,
+    Spreading,
+    SelfAssociation,
+    BiStericMassAction,
+    MultistateStericMassAction,
+    SimplifiedMultistateStericMassAction,
+    Saska,
+    GeneralizedIonExchange
+)
+
+
+def create_lwe(n_comp=4, unit_type=None, **kwargs) -> None:
+
+    def configure_all():
+        unit_types = [
+            'cstr', 'generalratemodel', 'tubularreactor',
+            'lumpedratemodelwithoutpores', 'lumpedratemodelwithpores', 'mct'
+        ]
+        for unit_type in unit_types:
+            configure_single_unit(unit_type)
+
+    def configure_single_unit(type_):
+        component_system = ComponentSystem(n_comp)
+        flow_sheet = FlowSheet(component_system)
+
+        if unit_type == 'cstr':
+            configure_cstr(component_system, kwargs)
+        elif unit_type == 'generalratemodel':
+            configure_grm(component_system, kwargs)
+        elif unit_type == 'tubularreactor':
+            configure_tb(component_system, kwargs)
```

Comment:
[observation] the tubular reactor cannot have particles / binding models. Would we still want to keep it here?

Reply:
Since it is a defined UnitOperation in CADET-process I included it to just get everything, but technically we already test this with the LRM. I am not sure what could happen that affects the Tubular Reactor but not the LRM, but If we can be sure that there is nothing I could take it out.